aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -19,6 +19,100 @@
 namespace at {
 namespace native {
 
+template<class scalar_t>
+void cusolver_LU(int m, int n, scalar_t* dA, int ldda, int* ipiv, int* info);
+
+template<class scalar_t>
+void cusolver_getrs(int n, int nrhs, scalar_t* dA, int lda, int* ipiv, scalar_t* ret, int ldb, int* info);
+
+template<class scalar_t>
+void cublas_LU_batched(int _m, int n, scalar_t** dA_array, int ldda, int* ipiv_array, int* info_array, int batchsize);
+
+template<class scalar_t>
+void cublas_getri_batched(int _m, int n, scalar_t** dA_array, int ldda, int* ipiv_array, int* info_array, int batchsize, scalar_t** dC_array);
+
+template<class scalar_t>
+static void _apply_single_inverse_helper(scalar_t* self_ptr, scalar_t* self_inv_ptr, int* ipiv_ptr, int* info_ptr, int n);
```

Comment:
This seems out of place, is that intentional? Otherwise for continuity, might be better to place above `apply_inverse`?

Reply:
Yeah, you are right. Will fix.